app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt

```diff
@@ -113,6 +88,20 @@ fun DashboardScreen(
         )
     }
 
+    val showManagerUpdateDialog by vm.prefs.showManagerUpdateDialog.getAsState()
+    var showDialog by rememberSaveable { mutableStateOf(showManagerUpdateDialog) }
+
+    if (showDialog) {
+        vm.updatedManagerVersion?.let { version ->
```

Comment:
This can be flattened using `derivedStateOf`. Remember to format it
```suggestion
    val availableUpdate by remember {
    	derivedStateOf { vm.updatedManagerVersion?.takeIf { showManagerUpdateDialog } }
    }

    availableUpdate?.let { version ->
```

Reply:
It's not dismissable then, how do you reassign availableUpdate to false onDismiss?

Reply:
You can dismiss it because `availableUpdate` will be null if `showManagerUpdateDialog` is false because of the `takeIf`. The `let` block will not run if `availableUpdate` is null

Reply:
`showManagerUpdateDialog` is a preference for the "Never show again", it is used on launch, should probably rename the preference to something like `showManagerUpdateDialogOnLaunch`

Reply:
My bad, it should have been `showDialog` instead of `showManagerUpdateDialog`. The code should work now